clang/tools/libclang/CXDiagnosticSetDiagnosticConsumer.h

```diff
@@ -0,0 +1,41 @@
+//===- CXDiagnosticSetDiagnosticConsumer.cpp - CXDiagnosticSet producing
+// consumer -===//
```

Comment:
I don't think we want to wrap the top line -- I would just drop the description part of it, or put it in the file comment below.